x-pack/plugins/enterprise_search/server/routes/app_search/engines.ts

```diff
@@ -60,4 +60,19 @@ export function registerEnginesRoutes({
       })(context, request, response);
     }
   );
+  router.get(
+    {
+      path: '/api/app_search/engines/{name}/overview',
```

Comment:
I mentioned this before, I'll mention it again to see what you think.

I don't particularly like renaming routes. If this forwards to `/overview_metrics` in Enterprise Search, I think it should also be `/overview_metrics` in the Kibana API.

It just simplifies things, IMO. One less thing to think about. If you see `/overview_metrics` throwing an error, it's simpler to just go look for the `/overview_metrics` API endpoint in the backend to debug, rather than first checking the server mapping to see what the real API endpoint is on the backend to debug.

That's my totally subjective opinion. Feel free to disregard and keep it as is. I don't think it ultimately will make a big difference one way or the other.

Reply:
That's a super fair point - what I was thinking originally when looking at this endpoint / view was:

1. What guarantees that our Overview page will only ever show metrics in the future? Why not show/pass other non-metric information?
2. If we ever do at that point, we would (likely?) change the name of the endpoint, and a generic `/overview` (to match the page) is more future-proof

That being said, I'm hoping we take a second pass at all our API endpoints and where possible convert internal ones to standard API endpoints in some distant future - at that point it would be great to look at names as well.